packages/tinacms/src/tina-cms.tsx

```diff
@@ -174,6 +199,8 @@ const parseURL = (url: string): { branch; isLocalClient; clientId } => {
   const pattern = new UrlPattern('/content/:clientId/github/:branch')
   const result = pattern.match(params.pathname)
 
+  // TODO if !result || !result.clientId || !result.branch, throw an error
```

Comment:
Locally, I can't recreate the infinite loop which leads me to think our login popup might be the culprit?  I'll defer to @logan-anderson , but I'm thinking maybe `edit-state` is recognizing the lack of a valid session and attempting to request one... which it can't do because the route isn't valid... which causes it to attempt to request one...

Reply:
Regardless, this feels like an edge case.  At the least, we spin off a card to address it specifically and let this go forward 👍